src-ts/proposalCreator.ts

```diff
@@ -225,15 +226,28 @@ export class RoundTripProposalCreator {
         l1Values.push(upgradeExecutorValue);
       }
     }
-
-    const l1TimelockScheduleCallData = l1Timelock.interface.encodeFunctionData("scheduleBatch", [
-      l1Targets,
-      l1Values,
-      l1CallDatas,
-      constants.HashZero,
-      descriptionHash,
-      minDelay,
-    ]);
+    let l1TimelockScheduleCallData: string;
```

Comment:
could use an IIFE and make this a const :)

Reply:
i thought you were anti-IIFE, maybe i'm misremembering tho, been awhile